style.css

```diff
@@ -103,3 +103,337 @@ body {
 li {
   padding: 15px;
 }
+
+/* middle section */
+
+section {
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  padding-left: 6.4%;
+  padding-right: 6.4%;
+  margin-bottom: 88px;
+}
+
+.works {
+  margin-bottom: 88px;
+}
+
+.single-card {
+  box-sizing: border-box;
+  display: flex;
+  padding-left: 24px;
+  padding-right: 24px;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 88px;
+}
```

Comment:
- [ ] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, add some spaces (Margin or Padding) between each card to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)👌

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/177019358-a91ebb13-0faa-46ea-9d5a-460294daea8f.png" width="100%" />  | <img src="https://user-images.githubusercontent.com/45231276/177019387-f11d7e49-9936-47ca-b9e0-b7309994cdf1.png" width="100%" /> |